src/components/principal/unit/resource/ResourceSummary.vue

```diff
@@ -20,28 +25,39 @@
 
 <script>
 
-  import Resource from './Resource.vue';
+import Resource from './Resource.vue'
 
-  export default {
-    name: 'ResourceSummary',
-    props: ['resource', 'rowNr'],
-    data: () => {
-      return {
-        showResourceDetails: false,
-      }
+export default {
+  name: 'ResourceSummary',
+  props: ['resource', 'rowNr'],
+  data: () => {
+    return {
+      crewSize: 0,
+      showResourceDetails: false
+    }
+  },
+  components: {
+    Resource
+  },
+  methods: {
+    mouseOver: function () {
+      this.showResourceDetails = true
     },
-    components: {
-      Resource,
+    mouseOut: function () {
+      this.showResourceDetails = false
     },
-    methods: {
-      mouseOver: function(){
-        this.showResourceDetails = true;
-      },
-      mouseOut: function(){
-        this.showResourceDetails = false;
-      },
+    calculateCrewSize: function () {
+      if (this.resource.crew != null) {
```

Comment:
I think is it enough if(this.resource.crew)   without explicit !== null

Reply:
Yes. Changed it and moved it in template